.github/CONTRIBUTING.md

```diff
@@ -144,6 +144,12 @@ make html
 
 and open `docs/build/html/index.html` in your browser.
 
+Notes:
+ - you need to have installed latex for rendering math equations. To do so you install TeXLive, you can do one of following:
+    * on linux run `apt install texlive`
+    * use [RTD docker image](https://hub.docker.com/r/readthedocs/build)
```

Comment:
```suggestion
 - You need to have LaTeX installed for rendering math equations. You can for example install TeXLive by doing one of the following:
    * on Linux run `apt install texlive` or otherwise follow the instructions on the TeXLive website
    * use the [RTD docker image](https://hub.docker.com/r/readthedocs/build)
```
btw not all Linux distros use apt, but yeah if someone uses a different OS they will know anyway how to do it with a different package manager :)